docs/literate/src/files/DGMulti_solver_mesh.jl

```diff
@@ -0,0 +1,220 @@
+                   coordinates_min=(-2.0, -2.0),
+                   coordinates_max=( 2.0,  2.0),
+                   periodicity=true)
+
+semi = SemidiscretizationHyperbolic(mesh, equations, initial_condition, dg,
+                                    boundary_conditions=boundary_condition_periodic)
+tspan = (0.0, 0.4)
+ode = semidiscretize(semi, tspan)
+
+alive_callback = AliveCallback(alive_interval=10)
+analysis_callback = AnalysisCallback(semi, interval=100, uEltype=real(dg))
+callbacks = CallbackSet(analysis_callback, alive_callback);
+
+# Run the simulation with the same time integration algorithm as before.
+sol = solve(ode, RDPK3SpFSAL49(), abstol=1.0e-6, reltol=1.0e-6,
+            callback=callbacks, save_everystep=false);
+#-
+using Plots
+pd = PlotData2D(sol)
+plot(pd)
+#-
+plot(pd["rho"])
+plot!(getmesh(pd))
+
+# This simulation is not as fast as the equivalent with `TreeMesh` since the functions are not
+# implemented that efficient. On the other hand, the functions are more general and thus we
+# have more option we can choose from.
+
+
+# ## Simulation with Gauss nodes
+# For instance, we can change the approximation type of our simulation.
+using Trixi, OrdinaryDiffEq
+equations = CompressibleEulerEquations2D(1.4)
+initial_condition = initial_condition_weak_blast_wave
+
+# We now use Gauss nodes instead of Gauss-Lobatto nodes which can be done for the element types
+# `Quad()` and `Hex()`. Therefore, we set `approximation_type=GaussSBP()`. Alternatively, we
+# can use a modal approach using the approximation type `Polynomial()`.
+dg = DGMulti(polydeg = 3,
+             element_type = Quad(),
+             approximation_type = GaussSBP(),
+             surface_flux = flux_lax_friedrichs,
+             volume_integral = VolumeIntegralFluxDifferencing(flux_ranocha))
+
+mesh = DGMultiMesh(dg,
+             cells_per_dimension=(32, 32), # initial_refinement_level = 5
+             coordinates_min=(-2.0, -2.0),
+             coordinates_max=( 2.0,  2.0),
+             periodicity=true)
+
+semi = SemidiscretizationHyperbolic(mesh, equations, initial_condition, dg,
+                              boundary_conditions=boundary_condition_periodic)
+tspan = (0.0, 0.4)
+ode = semidiscretize(semi, tspan)
+
+alive_callback = AliveCallback(alive_interval=10)
+analysis_callback = AnalysisCallback(semi, interval=100, uEltype=real(dg))
+callbacks = CallbackSet(analysis_callback, alive_callback);
+
+sol = solve(ode, RDPK3SpFSAL49(), abstol=1.0e-6, reltol=1.0e-6,
+            callback=callbacks, save_everystep=false);
+#-
+using Plots
+pd = PlotData2D(sol)
+plot(pd)
+
+
+# ## Simulation with triangular elements
+# Also, we can set another element type. We want to use triangles now.
+using Trixi, OrdinaryDiffEq
+equations = CompressibleEulerEquations2D(1.4)
+initial_condition = initial_condition_weak_blast_wave
+
+# We are using Gauss-Lobatto nodes again, so `approximation_type = SBP()`.
```

Comment:
Since there is no direct equivalent to Gauss-Lobatto nodes on triangles, it may be helpful to clarify that the `SBP()` approximation type on Triangles uses Gauss-Lobatto nodes on faces. More details can be found in https://jlchan.github.io/StartUpDG.jl/dev/RefElemData/#RefElemData-based-on-SBP-finite-differences.